addons/osfstorage/models.py

```diff
@@ -408,7 +408,7 @@ def save(self, skip_search=False, *args, **kwargs):
 
 class OsfStorageFolder(OsfStorageFileNode, Folder):
 
-    is_root = models.NullBooleanField()
```

Comment:
Changing these NullBooleanFields isn't required by Django 3, but it will be in Django 4 and it's very convenient to fix these and the `on_delete` behavior at the same time.